calcom/cal.com
# Add 'after event ends' trigger to workflows

## What does this PR do?

Adds 'after event ends' trigger to workflows.

Fixes #3708 ('Meeting ended' trigger for webhooks was already added with #3827)
Fixes #4785

<img width="869" alt="Screenshot 2022-10-06 at 16 20 22" src="https://user-images.githubusercontent.com/30310907/194337737-abd6ce82-f2a7-4f7b-80bb-a6012e851068.png">

**Environment**: Staging(main branch)

## Type of change


## How should this be tested?

- Create a new workflow with the new trigger and set it active for one event-type
- Book the corresponding event-type 
- Check in db if new workflowReminder was created
- Check in db if the scheduled date of the workflowReminder is set as expected

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
